sql/bank_test.go

```diff
@@ -26,16 +26,36 @@ import (
 	"testing"
 
 	"github.com/cockroachdb/cockroach/server"
+	cockroachSql "github.com/cockroachdb/cockroach/sql"
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
 var maxTransfer = flag.Int("max-transfer", 999, "Maximum amount to transfer in one transaction.")
 var numAccounts = flag.Int("num-accounts", 999, "Number of accounts.")
 
-// BenchmarkBank mirrors the SQL performed by examples/sql_bank, but structured
-// as a benchmark for easier usage of the Go performance analysis tools like
-// pprof, memprof and trace.
-func BenchmarkBank(b *testing.B) {
+// BenchmarkBankNoDescriptorCache mirrors the SQL performed by examples/sql_bank,
+// but structured as a benchmark for easier usage of the Go performance analysis
+// tools like pprof, memprof and trace.
+// It does not make use of the system cache.
+func BenchmarkBankNoDescriptorCache(b *testing.B) {
```

Comment:
If the benchmark has the same name, you can use [benchcmp](https://godoc.org/golang.org/x/tools/cmd/benchcmp) and [benchstat](https://github.com/rsc/benchstat) to create pretty before and after comparisons to include in your commit descriptions.